<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AIChat</class>
 <widget class="QWidget" name="AIChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="LeftPanel">
      <property name="styleSheet">
       <string notr="true">        QFrame {
            background: linear-gradient(135deg, #e3f2fd, #bbdefb);
            border-radius: 20px;
            //box-shadow: 0 4px 12px rgba(0, 0, 0, 0.1);
        }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>181</width>
         <height>601</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="LeftLayout" stretch="3,1,1,2">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="m_cartoonLabel">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 3px solid #90caf9;
            border-radius: 50%;
            background-color: white;
            padding: 10px;</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="m_assistantNameLabel">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1677</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 20px;
            font-weight: bold;
            color: #1565c0;
background-color: rgba(0,0,0,0)
            </string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;心语AI&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="statusLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Songti SC</family>
            <pointsize>-1</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">            font-size: 18px;
            font-weight: bold;
            color: #43a047;
            background-color: #e8f5e9;
            border-radius: 15px;
            padding: 5px 15px;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;在 线&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="chatFrame">
       <property name="geometry">
        <rect>
         <x>189</x>
         <y>10</y>
         <width>541</width>
         <height>581</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">           background-color: white;
            border-radius: 20px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>541</width>
          <height>581</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="chatLayout" stretch="15,2">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QTextEdit" name="m_chatDisplay">
           <property name="styleSheet">
            <string notr="true">            background-color: #f8f9fa;
            border-radius: 15px;
            padding: 15px;
            font-size: 14px;
            border: 1px solid #e0e7ff;
            line-height: 1.6;</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="inputLayout">
           <item>
            <widget class="QLabel" name="m_userAvatar">
             <property name="minimumSize">
              <size>
               <width>45</width>
               <height>45</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>45</width>
               <height>45</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">            border-radius: 50%;
            border: 2px solid #bbdefb;
            background-color: white;
            padding: 2px;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="m_inputBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"> background-color: #f0f4f8;
            border: 2px solid #c5d9f1;
            border-radius: 25px;
            padding: 12px 20px;
            font-size: 14px;
            color: #2d3748;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_sendButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 2px solid #4CAF50; 
    border-radius: 10px;              /* 圆角边框，可改为0表示*/
    background-color: rgb(143, 255, 129);      
font: 700 18pt &quot;Kaiti SC&quot;;

</string>
             </property>
             <property name="text">
              <string>发送</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
